tests/submission/test_endpoints.py

```diff
@@ -101,35 +98,31 @@ def put_tcga_brca(client, submitter):
     return r
 
 
-def test_program_creation_endpoint(client, pg_driver, submitter, dictionary_setup):
-    dictionary_setup('s3://test.com')
-    resp = put_cgci(client, auth=submitter)
+def test_program_creation_endpoint(client, pg_driver, admin):
+    resp = put_cgci(client, auth=admin)
     assert resp.status_code == 200, resp.data
     print resp.data
     resp = client.get('/v0/submission/')
     assert resp.json['links'] == ['/v0/submission/CGCI'], resp.json
 
 
-def test_program_creation_without_admin_token(client, pg_driver, submitter, dictionary_setup):
-    dictionary_setup('s3://test.com')
+def test_program_creation_without_admin_token(client, pg_driver, submitter):
```

Comment:
Looks like a mismatch of Roles. Was a member role, became a submitter role. I'm guessing admin == submitter from the above tests.

Reply:
the submitter fixture before was a bad name, it's basically a role generator function. 
The new fixtures are actual users with correct roles